libpod/container.go

```diff
@@ -414,6 +414,11 @@ type ContainerConfig struct {
 
 	// HealthCheckConfig has the health check command and related timings
 	HealthCheckConfig *manifest.Schema2HealthConfig `json:"healthcheck"`
+
+	// PreserveFDs is a number of additional file descriptors (in addition
+	// to 0, 1, 2) that will be passed to the executed process. The total FDs
+	// passed will be 3 + PreserveFDs.
+	PreserveFDs uint `json:"-"`
```

Comment:
Can you add a comment that this is not in the JSON because it is not supported for remote connections?

Reply:
Oh, wait, nevermind - this needs to actually have a JSON tag here, not `"-"` - that should only be in Specgen. This is for the DB, and we do want to save this to the container in the DB.